test-src/manageamiset_test.js

```diff
@@ -0,0 +1,107 @@
+var slugid = require('slugid');
```

Comment:
random nit, this file uses `let` and `var`.  We should probably chose one and stick to it.

The difference in case you're unfamiliar is that var is in scope through the entire function.  If you use var in an `if`, the variable is in scope through the entire function.  Let is block scope, if you use let in an `if`, it's only in scope inside that `if` and those blocks (e.g. code inside of `{ }`) that are nested.

Let is better than var!  Also, if you're really curious do a search about 'variable hoisting' for more info on how var works.